includes/ui/class-ui-elements.php

```diff
@@ -0,0 +1,176 @@
+<?php
+/**
+ * Notice component for the frontend.
```

Comment:
I think this "Notice" comment is not relevant to what the class does. It seems to be encapsulating various UI utils and elements.